server/db/migrations/20_create_grid_service_instance.rb

```diff
@@ -1,7 +1,12 @@
 class CreateGridServiceInstance < Mongodb::Migration
   def self.up
     GridServiceInstance.create_indexes
-    Container.unscoped.where(container_type: 'container').includes(:grid_service).each do |c|
+    migrate_containers('volume')
+    migrate_containers('container')
```

Comment:
> And I don't really understand why this is a problem? :)

I'm just unsure how it works exactly.  If there's multiple `volume` containers for the same instance on different nodes, but only one `container` type container, then this migration change could move the stateful container from the host node where it is currently running to some other host node where it used to be running?